### ISSUE TITLE
DataInspector broken with 0.17.7

### ISSUE THREAD

**Issue by tecosaur** [JuliaPlots/Makie.jl#2076]
Hi, I've just upgraded and am trying to use the `DataInspector`. Unfortunately, it doesn't seem like it wants to be used :sweat_smile:.

```julia
julia> using GLMakie

julia> fig = Figure();

julia> ax = Axis(fig[1,1]);

julia> DataInspector(ax)
ERROR: type Array has no field extents
Stacktrace:
  [1] getproperty
    @ ./Base.jl:42 [inlined]
  [2] Bbox_from_glyphcollection(text::String, gc::Vector{Point{2, Float32}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:191
  [3] (::Makie.var"#883#886"{Observable{Any}})(gc::Vector{Point{2, Float32}}, pos::Point{2, Float32}, pad::Vec{4, Float32})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:294
  [4] #map#19
    @ ~/.julia/packages/Observables/ynr7h/src/Observables.jl:444 [inlined]
  [5] map(::Makie.var"#883#886"{Observable{Any}}, ::Observable{Vector{Point{2, Float32}}}, ::Observable{Any}, ::Observable{Any})
    @ Observables ~/.julia/packages/Observables/ynr7h/src/Observables.jl:444
  [6] plot!(plot::Combined{Makie._inspector, Tuple{Int64}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:293
  [7] plot!(scene::Scene, P::Type{Combined{Makie._inspector, Tuple{Int64}}}, attributes::Attributes, input::Tuple{Observable{Int64}}, args::Observable{Tuple{Int64}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:402
  [8] plot!(scene::Scene, P::Type{Combined{Makie._inspector}}, attributes::Attributes, args::Int64; kw_attributes::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:320
  [9] plot!
    @ ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:288 [inlined]
 [10] #plot!#132
    @ ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:271 [inlined]
 [11] _inspector!(::Scene, ::Vararg{Any}; attributes::Base.Pairs{Symbol, Observable{StaticArrays.SMatrix{4, 4, Float32, 16}}, Tuple{Symbol}, NamedTuple{(:_root_px_projection,), Tuple{Observable{StaticArrays.SMatrix{4, 4, Float32, 16}}}}})
    @ Makie ~/.julia/packages/MakieCore/4Hq26/src/recipes.jl:37
 [12] DataInspector(scene::Scene; priority::Int64, kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:442
 [13] DataInspector
    @ ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:439 [inlined]
 [14] #DataInspector#889
    @ ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:435 [inlined]
 [15] DataInspector(fig_or_block::Axis)
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:435
 [16] top-level scope
    @ REPL[4]:1
```

**Comment by fbanning** [comment]
Can confirm this bug.

```julia
# Makie 0.17.7 and GLMakie 0.6.7

julia> scatter(1:10)

julia> DataInspector()
ERROR: type Array has no field extents
Stacktrace:
  [1] getproperty
    @ ./Base.jl:42 [inlined]
  [2] Bbox_from_glyphcollection(text::String, gc::Vector{Point{2, Float32}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:191
  [3] (::Makie.var"#883#886"{Observable{Any}})(gc::Vector{Point{2, Float32}}, pos::Point{2, Float32}, pad::Vec{4, Float32})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:294
  [4] #map#19
    @ ~/.julia/packages/Observables/ynr7h/src/Observables.jl:444 [inlined]
  [5] map(::Makie.var"#883#886"{Observable{Any}}, ::Observable{Vector{Point{2, Float32}}}, ::Observable{Any}, ::Observable{Any})
    @ Observables ~/.julia/packages/Observables/ynr7h/src/Observables.jl:444
  [6] plot!(plot::Combined{Makie._inspector, Tuple{Int64}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:293
  [7] plot!(scene::Scene, P::Type{Combined{Makie._inspector, Tuple{Int64}}}, attributes::Attributes, input::Tuple{Observable{Int64}}, args::Observable{Tuple{Int64}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:402
  [8] plot!(scene::Scene, P::Type{Combined{Makie._inspector}}, attributes::Attributes, args::Int64; kw_attributes::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:320
  [9] plot!
    @ ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:288 [inlined]
 [10] #plot!#132
    @ ~/.julia/packages/Makie/fJ4OE/src/interfaces.jl:271 [inlined]
 [11] _inspector!(::Scene, ::Vararg{Any}; attributes::Base.Pairs{Symbol, Observable{StaticArrays.SMatrix{4, 4, Float32, 16}}, Tuple{Symbol}, NamedTuple{(:_root_px_projection,), Tuple{Observable{StaticArrays.SMatrix{4, 4, Float32, 16}}}}})
    @ Makie ~/.julia/packages/MakieCore/4Hq26/src/recipes.jl:37
 [12] DataInspector(scene::Scene; priority::Int64, kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:442
 [13] DataInspector
    @ ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:439 [inlined]
 [14] #DataInspector#889
    @ ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:435 [inlined]
 [15] DataInspector
    @ ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:435 [inlined]
 [16] #DataInspector#895
    @ ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:458 [inlined]
 [17] DataInspector()
    @ Makie ~/.julia/packages/Makie/fJ4OE/src/interaction/inspector.jl:458
 [18] top-level scope
    @ REPL[4]:1
```

Happy to see that it's already being worked on. Thanks, @ffreyer . :)

**Comment by ffreyer** [comment]
Should be fixed in 0.17.8

**Comment by tecosaur** [comment]
Great!
